images/scanner-test.Dockerfile

```diff
@@ -49,7 +48,6 @@ RUN dnf install -y \
         lsof \
         lz4 \
         openssl \
-        postgresql${PG_MAJOR}-server \
```

Comment:
why is this removed? It seems like postgres is removed from the image, and it's needed for DB testing

Reply:
I was testing something. Good catch